src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx

```diff
@@ -45,7 +41,7 @@ Simple arrays are used to store a list of values of the same primitive type; for
 
     `getfield(durations, 2)` will return `90`.
 
-    An alternative to the function `getfield()` is to use the square brackets `[ ]` shorthand.
+    Alternatively, you can use the square brackets `[ ]` shorthand instead of the function `getfield()`.
```

Comment:
I think you can just drop "alternatively" completely and other filler words like that